application/controllers/AjaxAlertController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * Controller for rendering AlertWidget via Ajax call
+ */
+class AjaxAlertController extends LSBaseController
```

Comment:
Admin only? Should have access rules?

Reply:
New code should/could/must be added to Psalm too, error level 2 preferably.

Reply:
Imo access rules not needed, it just spits out html, possibly usable for surveytaking, too.